src/Microsoft.Diagnostics.Runtime/src/Common/IAddressableTypedEntity.cs

```diff
@@ -0,0 +1,28 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+
+namespace Microsoft.Diagnostics.Runtime
+{
+    /// <summary>
+    /// Represents an addressable entity (class or struct) with associated type.
+    /// </summary>
+    public interface IAddressableTypedEntity: IEquatable<IAddressableTypedEntity>
+    {
+        ulong Address { get; }
```

Comment:
Can you copy/paste the documentation strings out of ClrObject onto the properties/methods in this interface?

Reply:
I added code documentation slightly changed from ClrObject.

The difference is how exceptions are documented - I used to [follow MS guides](https://docs.microsoft.com/en-us/dotnet/csharp/codedoc) to let intelisence/resharper pick exceptions.

Please let me know if you prefer differently.

Thanks.